bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -178,6 +178,39 @@ spec:
             command: ["test", "-f", "/etc/cni/net.d/80-openshift-network.conf"]
           initialDelaySeconds: 5
           periodSeconds: 5
+{{- if .EnableConnectivityChecks }}
+      - name: sdn-check-endpoints
+        image: quay.io/openshift/origin-cluster-kube-apiserver-operator:latest
```

Comment:
obviously this needs to be changed to the real image reference when this merges.

Reply:
For sure

Reply:
https://github.com/openshift/cluster-network-operator/pull/872

Reply:
per @squeed , doesn't matter to split, thus I closed #872 and just added cluster-network-operator image to deployments.